crates/sui-core/src/authority.rs

```diff
@@ -884,21 +885,19 @@ impl AuthorityState {
         self.commit_certificate(inner_temporary_store, certificate, effects, epoch_store)
             .await?;
 
-        // Notifies transaction manager about available input objects. This allows the transaction
-        // manager to schedule ready transactions.
-        //
-        // REQUIRED: this must be called after commit_certificate() (above), to ensure
-        // TransactionManager can receive the notifications for objects that it did not find
-        // in the objects table.
-        //
-        // REQUIRED: this must be called before tx_guard.commit_tx() (below), to ensure
```

Comment:
ooc why is this no longer true?

Reply:
tx_guard.commit_tx() is currently a no-op FYI

Reply:
Right, and the existence of executed effects are checked when transaction manager recovers from `pending_execution` table now, so even if a txn did not call tx_guard.commit_tx() due to a crash gets retried, transaction manager would handle the case correctly by avoiding re-execution and removing the transaction from the `pending_execution` table.